docs/releases/3.0.txt

```diff
@@ -330,8 +330,7 @@ Security
   uses frames of itself, you will need to explicitly set ``X_FRAME_OPTIONS =
   'SAMEORIGIN'`` for them to continue working.
 
-* :setting:`SECURE_CONTENT_TYPE_NOSNIFF` setting now defaults to ``True``. With
-  the enabled :setting:`SECURE_CONTENT_TYPE_NOSNIFF`, the
+* :setting:`SECURE_CONTENT_TYPE_NOSNIFF` now defaults to ``True``. This means
```

Comment:
I think the (probably intended) `With this enabled...` is marginally better than `This means...`